test_podcastparser.py

```diff
@@ -21,8 +21,11 @@
 import os
 import glob
 import json
+from StringIO import StringIO
```

Comment:
Yeah, I'd already seen that. I love automated testing! :)

However, from what I can tell the imports have to be the other way round, as `io.StringIO` _does_ exist in Python 2.7, but doesn't behave quite the same way as the legacy `StringIO.StringIO`.